### PR TITLE
DDP-4898: [TestBoston] Help page

### PR DESCRIPTION
**Pull request by fat23cat**
Mockup:
image

### CODE REVIEW

File: ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <header *ngIf="!isAdmin" class="header" [ngClass]="{'header_shadow': isPageScrolled && !isMenuOpened}">
     <div class="header-content">
-        <a class="project-logo" routerLink="/">
+        <a class="project-logo" routerLink="/" (click)="closeMenu()">
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html:3]: Good idea!

**Review by MocanaAtBroad — COMMENTED** [review]
A couple of suggestions:

**Review by MocanaAtBroad — COMMENTED** [review]
I think would be better if this link opens in separate tab so our page can be used as reference:
Screen Shot 2020-09-08 at 5 24 46 PM

**Review by MocanaAtBroad — COMMENTED** [review]
Would be better if the text "location type" was capitalized and bolded so it echoes the text in the picture.
Also would be nice if we had the search icon Screen Shot 2020-09-08 at 5 32 23 PM in the instructions text.


Screen Shot 2020-09-08 at 5 20 18 PM

**Comment by fat23cat** [comment]
> I think would be better if this link opens in separate tab so our page can be used as reference:

Hm, I've double-checked, when I click "www.ups.com/dropoff" it opens in the separate tab.

> Would be better if the text "location type" was capitalized and bolded so it echoes the text in the picture.

Good catch, thanks!

> Also would be nice if we had the search icon Screen Shot 2020-09-08 at 5 32 23 PM in the instructions text.

Nice suggestion, but I have to discuss it with Jen and Erin.

**Comment by fat23cat** [comment]
> Would be better if the text "location type" was capitalized and bolded so it echoes the text in the picture.

Good catch, thanks!

**Comment by fat23cat** [comment]
> Also would be nice if we had the search icon Screen Shot 2020-09-08 at 5 32 23 PM in the instructions text.

Nice suggestion, but I need to discuss it with Jen and Erin.

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**